go.sum

```diff
@@ -1,3 +1,6 @@
+github.com/ably/ably-go v1.2.0/go.mod h1:bdOnFcqJrbZE7WSp1jqbdsIMBo5aMGXh98GWiiwmeZ0=
```

Comment:
It seems odd that this introduces a circular dependency now, what benefit is there in moving `ablytest` to a separate repository?